cfme/common/provider_views.py

```diff
@@ -572,9 +573,11 @@ class InfraProviderEditView(ProviderEditView):
     """
     @property
     def is_displayed(self):
+        expected_title = ("Edit Infrastructure Providers '{name}'".format(
```

Comment:
Question: This works for 5.10 and 5.9 versions?

Reply:
Yes, it works for both 5.10 and 5.9 versions.

Reply:
Sooo, I'd like you to investigate this
**REQUIRED:** Inside each of these classes there is a ```string_name``` which is going to be either ```Cloud``` or ```Infrastructure``` allowing you to do ```self.context['object'].string_name``` as a replacement for the static strings above. This means you could almost push this definition up into the common file. Please experiment and see if it works.